src/public/scss/_wg.scss

```diff
@@ -43,8 +42,29 @@ $whitepurple: #eed5e3;
     line-height: 24px;
   }
 
-  h1, h2, h3, h4, h5, h6, p, ol, ul, a, table, .govuk-header__link, .govuk-task-list__status, .govuk-tag, .govuk-input, .govuk-label, .govuk-button, .govuk-select, dl, .govuk-accordion__section-heading-text, span, .govuk-inset-text {
-    font-family: Arial, Helvetica, "Nimbus Sans L", sans-serif;
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6,
+  p,
+  ol,
+  ul,
+  a,
+  table,
+  .govuk-header__link,
+  .govuk-task-list__status,
+  .govuk-tag,
+  .govuk-input,
+  .govuk-label,
+  .govuk-button,
+  .govuk-select,
+  dl,
+  .govuk-accordion__section-heading-text,
+  span,
+  .govuk-inset-text {
+    font-family: Arial, Helvetica, 'Nimbus Sans L', sans-serif;
```

Comment:
Your editor prefs are leaking :D

Reply:
yeah is this a good thing or bad thing? My editor uses prettierconfig and editorconfig, so this is the configured style

Reply:
I'll rebase with the prettier changes once merged in